resources/report-templates/standard-template/jinja2/analysis-detail.html

```diff
@@ -1,3 +1,4 @@
+{% if detail_table %}
```

Comment:
Safely create empty string if the layer doesn't support analysis breakdown (like population)